locale/model_attributes.rb

```diff
@@ -267,58 +267,34 @@
 # TRANSLATORS: "Table name" or "Table name|Column name" for error messages
 _('Variable lookup key')
 # TRANSLATORS: "Table name" or "Table name|Column name" for error messages
-_('VariableLookupKey|Avoid duplicates')
+_('Avoid duplicates')
```

Comment:
These should be `Table name|Colum name` per the comment, i.e. `LookupKey|Avoid duplicates` or they won't match the automatic string lookups.